core/templates/dev/head/pages/exploration_editor/preview_tab/preview_tab.html

```diff
@@ -32,7 +32,7 @@
   .oppia-preview-bottom-right-container {
     bottom: 5px;
     display: block;
-    position: absolute;
+    position: fixed;
```

Comment:
Why fixed positioning?
I mean is it working when the learner page have scrollable length? 
(I'll do the manual test once, Idk why my develop branch, is falling downloading some third_party dependencies)
Also, check when we have supplement cards! (Like image interaction)

FYI, currently, I can see in prod site refresh button use to be at the end of the preview tab (take that as reference).